### PR TITLE
created PR beacause of formatted with black formatter

### PR DESCRIPTION
**Pull request by tonyxuan2021**
-  formatted codewith black formatter, coding change will be coming in the next PR.

### CODE REVIEW

File: handler/driver_get_route.py [hunk 1]
```diff
@@ -4,25 +4,28 @@
 from modals.Drivers import Users
 from config import SECRET_KEY
 
-driver_route = Blueprint('driver_route', __name__)
+driver_route = Blueprint("driver_route", __name__)
```
Comment (chunloy) [inline handler/driver_get_route.py:7]: We decided not to use Blueprint, correct? We could change this in another PR

**Review by chunloy — APPROVED** [review]
Looks good

**Review by yhus — APPROVED** [review]
👌

**Review by StudentCZ — APPROVED** [review]
I received clarification from the Tony. Everything looks good.

File: handler/driver_get_route.py (hunk 1 above)
Comment (tonyxuan2021) [reply handler/driver_get_route.py:7]: Correct. I will remove blueprint in my next PR.

**Pull request merged**